content/3-lifecycle/2-on-unmount/react/Time.jsx

```diff
@@ -11,7 +11,7 @@ export default function Time() {
 		return () => {
 			clearInterval(timer);
 		};
-	});
+	}, []);
```

Comment:
You don't need to specify an empty array of dependency for this use-case.
https://reactjs.org/docs/hooks-effect.html#recap